### PR TITLE
Usage of denyAccessUnlessGranted in the controller

### PR DESCRIPTION
**Pull request by 94noni**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to | >= 2.6 |
| Fixed tickets | - |

Just a mini patch

### CODE REVIEW

File: cookbook/security/voters_data_permission.rst [hunk 1]
```diff
@@ -213,9 +212,7 @@ from the authorization checker is called.
             $post = ...;
 
             // keep in mind, this will call all registered security voters
-            if (false === $this->get('security.authorization_checker')->isGranted('view', $post)) {
-                throw new AccessDeniedException('Unauthorised access!');
-            }
+            $this->denyAccessUnlessGranted('view', $post, 'Unauthorised access!');
```
Comment (javiereguiluz) [inline cookbook/security/voters_data_permission.rst]: Minor comment: I know that the original doc said `Unauthorised`, but we use the American English dialect in the Symfony docs, so this should be changed by `Unauthorized`. Thanks!

Comment (javiereguiluz) [inline cookbook/security/voters_data_permission.rst]: When using shortcuts, we usually show the long/boring alternative in commented code:

```php
    $post = ...;

    // keep in mind, this will call all registered security voters
    $this->denyAccessUnlessGranted('view', $post, 'Unauthorised access!');

    // the equivalent code without using the denyAccessUnlessGranted() shortcut
    // use Symfony\Component\Security\Core\Exception\AccessDeniedException;
    //
    // if (false === $this->get('security.authorization_checker')->isGranted('view', $post)) {
    //     throw new AccessDeniedException('Unauthorised access!');
    // }
```

File: cookbook/security/voters_data_permission.rst [hunk 2]
```diff
@@ -224,5 +228,8 @@ from the authorization checker is called.
 .. versionadded:: 2.6
     The ``security.authorization_checker`` service was introduced in Symfony 2.6. Prior
     to Symfony 2.6, you had to use the ``isGranted()`` method of the ``security.context`` service.
+    
+    ``denyAccessUnlessGranted()`` method was introduced in Symfony 2.6 as a shortcut.
```
Comment (WouterJ) [inline cookbook/security/voters_data_permission.rst]: I remember having display issues on symfony.com with blank lines in versionadded directives. Can you please start a new one (repeat line 228 before this line)?

**Comment by WouterJ** [comment]
Thanks, love it! Can you please fix my minor issue, I'll merge it in after it.

**Comment by 94noni** [comment]
ping @WouterJ is it ok?

File: cookbook/security/voters_data_permission.rst [hunk 3]
```diff
@@ -224,5 +228,8 @@ from the authorization checker is called.
 .. versionadded:: 2.6
     The ``security.authorization_checker`` service was introduced in Symfony 2.6. Prior
     to Symfony 2.6, you had to use the ``isGranted()`` method of the ``security.context`` service.
+.. versionadded:: 2.6
```
Comment (xabbuh) [inline cookbook/security/voters_data_permission.rst]: Better add a blank line before this.

File: cookbook/security/voters_data_permission.rst [hunk 4]
```diff
@@ -224,5 +228,8 @@ from the authorization checker is called.
...
+    ``denyAccessUnlessGranted()`` method was introduced in Symfony 2.6 as a shortcut.
```
Comment (xabbuh) [inline cookbook/security/voters_data_permission.rst]: I would reword this a bit: "The `denyAccessUnlessGranted()`method [...]"

File: cookbook/security/voters_data_permission.rst [hunk 5]
```diff
@@ -224,5 +228,8 @@ from the authorization checker is called.
...
+    This uses ``security.authorization_checker`` and throws ``AccessDeniedException`` if needed.
```
Comment (xabbuh) [inline cookbook/security/voters_data_permission.rst]: It uses `security.authorization_checker` and throws an `AccessDeniedException` if needed.

**Comment by 94noni** [comment]
@WouterJ ping

**Comment by WouterJ** [comment]
@94noni it looks great now. Thanks for the head-up!

**Comment by WouterJ** [comment]
Good catch, thanks Antoine.
